walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -18,10 +18,34 @@
  * Utility class for creating greeting messages.
  */
 public class Greeter {
+
+  /**
+   * Return the input with leading whitespace removed.
+   */
+  private StringBuilder removeSpaceFromFront(StringBuilder sb) {
+    while (sb.length() > 0 && sb.charAt(0) == ' ') {
+        sb.deleteCharAt(0);
+    }
+    return sb;
+  }
+
   /**
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    StringBuilder sb = new StringBuilder();
+    sb.append(name);
+    sb = removeSpaceFromFront( removeSpaceFromFront(sb).reverse() ).reverse();
```

Comment:
I think you can just use sb.append(name.strip()).

Reply:
Good to know

Reply:
Looks like trim() does the same thing since strip() is not available in Java 8